src/commands/a32nx/beginner-guide.ts

```diff
@@ -9,7 +9,22 @@ export const beginner: CommandDefinition = {
     executor: async (msg) => {
         const beginnerEmbed = makeEmbed({
             title: 'FlyByWire A32NX | Beginner\'s Guide',
-            description: 'Please see our [beginner\'s guide](https://docs.flybywiresim.com/pilots-corner/beginner-guide/overview/) for information on how to set up and fly the A32NX.',
+            description: makeLines ([
+                'Please see our [beginner\'s guide](https://docs.flybywiresim.com/pilots-corner/beginner-guide/overview/) for information on how to set up and fly the A32NX.',
+                'It contains detailed information for both new sim pilots and veterans looking for a refresher on certain topics.',
```

Comment:
```suggestion
                'Please see our [beginner\'s guide](https://docs.flybywiresim.com/pilots-corner/beginner-guide/overview/) for information on how to set up and fly the A32NX. '
                +'It contains detailed information for both new sim pilots and veterans looking for a refresher on certain topics.',
```

Small formatting thing for different window sizes. :)